pallets/phala-world/src/incubation.rs

```diff
@@ -441,8 +442,7 @@ pub mod pallet {
 				Origin::<T>::Signed(owner.clone()).into(),
 				collection_id,
 				nft_id,
-				1,
-			)?;
+			).map_err(|e| e.error)?;
```

Comment:
@HashWarlock I'm not sure my change is correct, could you help me fix this?

Reply:
If not do like this, it will compile error
```
error[E0277]: `?` couldn't convert the error to `sp_runtime::DispatchError`
   --> pallets/phala-world/src/incubation.rs:445:5
    |
445 |             )?;
    |              ^ the trait `From<DispatchErrorWithPostInfo<PostDispatchInfo>>` is not implemented for `sp_runtime::DispatchError`
    |
```

Reply:
I'll create a branch off and start fixing these changes